src/index.html

```diff
@@ -31,11 +27,21 @@
           align-items: center;
           font-size: 30px;
           color: #fff;
-          flex-direction: column;">
+          flex-direction: column;">Loading
         </div>
       </div>
     </div>
+    
     <script src="//code.jquery.com/jquery-1.12.0.min.js"></script>
+    <script>
+       $("head").append('<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.2/css/font-awesome.min.css" rel="stylesheet" />');
+       $("head").append('<link href="/css/bootstrap.css" rel="stylesheet" />');
+       $("head").append('<link href="//cdn.bootcss.com/bootstrap-social/4.12.0/bootstrap-social.min.css" rel="stylesheet" />');
+       $("head").append('<link href="//cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" rel="stylesheet"/>');
+       $("head").append('<link href="//cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick.min.css" rel="stylesheet"/>');
+       $("head").append('<link href="//cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.6.0/slick-theme.min.css" rel="stylesheet" />');
+       $("head").append('<link href="/css/main.css" rel="stylesheet" />');
+      </script>
```

Comment:
@Anoninnyc is there a reason you use multiple '$("head").append' entries instead of a single append?
Also, you repeat the append of "slick.min.css" twice.